src/core/TimePos.cpp

```diff
@@ -23,6 +23,7 @@
  *
  */
 
+#include <cassert>
```

Comment:
This should be included after `TimePos.h`. (The first inclusion in any source file should be the corresponding header file.)